doc/OVERMAP.md

```diff
@@ -0,0 +1,426 @@
+* a road crossing a river will be a bridge
+
+The **overmap_connection** data will then be used to create a linear road feature connecting two points,
+applying the previously defined rules.
+
+### overmap_location
+
+We've previously mentioned defining valid overmap terrain types for the placement of overmap specials,
+city buildings, and overmap connections, but one thing to clarify is these actually leverage another
+type called an **overmap_location** rather than referencing **overmap_terrain** values directly.
+
+Simply put, an **overmap_location** is just a named collection of **overmap_terrain** values. 
+
+For example, here are two simple definitions.
+
+```json
+{
+    "type": "overmap_location",
+    "id": "forest",
+    "terrains": ["forest"]
+}, 
+{
+    "type": "overmap_location",
+    "id": "wilderness",
+    "terrains": ["forest", "field"]
+}
+```
+
+The value of these is that they allow for a given overmap terrain to belong to several different
+locations and provide a level of indirection so that when new overmap terrains are added (or
+existing ones removed), only the relevant **overmap_location** entries need to be changed rather
+than every **overmap_connection**, **overmap_special**, and **city_building** that needs those
+groups.
+
+For example, with the addition of a new forest overmap terrain `forest_thick`, we only have to
+update these definitions as follows:
+
+```json
+{
+    "type": "overmap_location",
+    "id": "forest",
+    "terrains": ["forest", "forest_thick"]
+},
+{
+    "type": "overmap_location",
+    "id": "wilderness",
+    "terrains": ["forest", "forest_thick", "field"]
+}
+```
+
+## Overmap Terrain
+
+### Rotation
+
+If an overmap terrain can be rotated (i.e. it does not have the `NO_ROTATE` flag), then when
+the game loads the definition from JSON, it will create the rotated definitions automatically, 
+suffixing the `id` defined here with `_north`, `_east`, `_south` or `_west`. This will be
+particularly relevant if the overmap terrains are used in **overmap_special** or **city_building**
+definitions, because if those are allowed to rotate, it's desirable to specify a particular 
+rotation for the referenced overmap terrains (e.g. the `_north` version for all).
+
+### Fields
+
+|    Identifier     |                                           Description                                            |
+| ----------------- | ------------------------------------------------------------------------------------------------ |
+| `type`            | Must be "overmap_terrain".                                                                       |
+| `id`              | Unique id.                                                                                       |
+| `name`            | Name for the location shown in game.                                                             |
+| `sym`             | Symbol used when drawing the location. ASCII (e.g. F is 70) plus some specials for line drawing. |
```

Comment:
Add list of line drawing symbols?

Note that certain symbols (v><^) are getting rotated according to overmap direction?

Mention that not all ASCII symbols are supported?

Reply:
`overmap_terrain` attributes are the ones I'm least familiar with. I think if someone else wants to write that content, by all means. It won't fit nicely in the table, but can be in a section below as I've done to elaborate on the others.